content/QRadar-SIEM/101.es.md

```diff
@@ -0,0 +1,270 @@
+---
+title: '101: Configuración de demostración de QRadar'
+timeToComplete: 30
+updated: 2023-06-12
+---
+
+# Guía de preparación de la demostración
+
+## Introducción
+
+En este laboratorio, usted configurará una instancia de demostración del producto QRadar Security Information and Event Management con datos de muestra. Después, el ambiente puede ser usado para explorar las características clave del producto o para entregar una demostración a un cliente. Aquí están los enlaces complementarios para que usted pueda utilizar **después** de completar el taller en persona.
+
+- Vea el [vídeo de ejemplo](https://ibm.seismic.com/Link/Content/DCDJ8qCbhJPB28MTDjXBpB63WP7d)que muestra cómo puede realizarse esta demostración.
+- Solicite una nueva demostración en vivo utilizando la [Instancia de QRadar reservada en la Zona Técnica](https://ibm.biz/L300-SEIM-TZ).
+- Prepare la demostración como se describe en esta [guía de preparación de demostraciones](/qradar-siem/102.md).
+- [OPCIONAL] Utilice la [demo virtual, también conocida como demo click-through,](https://ibm.seismic.com/Link/Content/DCMh7JWdbXjQR89MR2TRjBJmPhJG)para este caso de uso.
+
+### Requisitos previos
+
+Aunque no es obligatorio, para obtener la mejor experiencia, se recomienda encarecidamente el uso de una VPN con [Wireguard](https://www.wireguard.com/). Por favor, instale esta aplicación en su estación de trabajo antes de completar el intento de conectarse a QRadar utilizando la URL directa o ssh directo. Si está utilizando una plataforma Microsoft Windows 10 o posterior, incluye un cliente ssh. De lo contrario, puede instalar [putty](https://www.putty.org/) o [git bash](https://gitforwindows.org/) para el acceso ssh directo.
+
+### \[OPCIONAL] Solicitar acceso a un despliegue en directo
+
+Más adelante, si desea demostrar el caso de uso utilizando el sistema en vivo, puede solicitar acceso al sistema de demostración de QRadar SIEM en la Zona Tecnológica de IBM. Para el taller presencial, estos pasos ya se han realizado por usted, así que pase a la [siguiente sección](#learn-about-lab-architecture-and-get-access)
+
+TODO - actualizar capturas de pantalla, TechZone ha cambiado
```

Comment:
I'm going to remove this from the canonical english version, the screen shots are close and I'll fix them soon.